src/main/java/com/google/sps/data/ProjectMetaData.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+/** Metadata for an individual project. */
+public class ProjectMetaData {
+
+  public float averageIAMBindingsInPastYear;
+
+  /** 
+   *  Create metadata that speicfy that this project had an average of
+   *  {@code averageIAmBindingsInPastYear} IAM Bindings per day in the past 
+   *  year. 
+   */
+  public ProjectMetaData(float averageIAMBindingsInPastYear) {
+    this.averageIAMBindingsInPastYear = averageIAMBindingsInPastYear;
+  }
+}
```

Comment:
Missing newline at the end of file

Reply:
It's not missing---there would be a little red circle with a slash if it were